src/main/scala/zio/ftp/Settings.scala

```diff
@@ -0,0 +1,98 @@
+package zio.ftp
+
+import java.net.Proxy
+
+import net.schmizz.sshj.{ Config => SshConfig, DefaultConfig => DefaultSshConfig }
+import net.schmizz.sshj.sftp.{ SFTPClient => JSFTPClient }
+import org.apache.commons.net.ftp.{ FTPClient => JFTPClient }
+import zio.Chunk
+
+sealed trait Settings[+A]
```

Comment:
I'd call this `FtpSettings`, because `Settings` is a very common name that is likely to clash.